### PR TITLE
Responsive navbar

### PR DESCRIPTION
**Pull request by VEBERArnaud**
## Description

This PR makes the snowshoe navbar:
- responsive
- fixed to top

### CODE REVIEW

File: views/pages/login.jsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ module.exports = React.createClass({
             <h2>Welcome to <strong>SnowShoe</strong> Dashboards</h2>
             <p>One must be connected before having access to his pull requests!</p>
             <p className="clearfix">
-              <Login className="btn-lg pull-right" href="/auth/github" role="button" />
+              <a href="/auth/github" className="btn btn-success btn-lg pull-right">Log In</a>
```
Comment (ludovic-gonthier) [inline views/pages/login.jsx]: You should use the `<Login>` component here

**Comment by ludovic-gonthier** [comment]
I am not on my laptop now, I can not test it for now, but the code seems fine :ok_hand: .
Just need to see the output

**Comment by ludovic-gonthier** [comment]
You can delete the `PR to watch` element finally! This way, the navbar is fully responsive (some glitches appears between 700-900px)

**Comment by VEBERArnaud** [comment]
Done

Do you want the navbar to be fluid (full-width) ?
I think it will give a better feel to the navbar with the higher resolutions

Edit: you can try with the web-inspector, just replace the `class="container"` with `class="container-fluid"` in the navbar

**Comment by ludovic-gonthier** [comment]
Let's give it a try!

In mobile mode, I found that the rate limit is still to the right of the screen and is at the bottom of the 'Log out' button. Is there a way for the rate to be on top of it instead and of the left side?

**Comment by VEBERArnaud** [comment]
nice catch, I'm trying to fix the rate limit glitch

File: views/components/dropdowns/teams.jsx [hunk 2]
```diff
@@ -16,11 +16,6 @@ module.exports = React.createClass({
   },
   render: function () {
     var dropdown;
-    var attributes = {};
```
Comment (ludovic-gonthier) [inline views/components/dropdowns/teams.jsx:19]: Why did you removed this part of the code?
It handles the disabling of the teams buttons when no Organization is selected.

**Comment by VEBERArnaud** [comment]
I removed it, because it was not working with the navbar dropdown. 

I didn't want to remove this feature in the first place, the possibilities of the bootstrap navbar made me do it.

We can improve the system to make it work in this case, but I need your react experience to make it "clean".

**Comment by ludovic-gonthier** [comment]
Ok,

I'll see what I can do. Maybe we could just bind the component on a window.resive event and affect the disabled just on big width device?

**Comment by VEBERArnaud** [comment]
Actually this is not a responsive issue, I tried to use the `class="disabled"` attribute on the link triggering the dropdown opening, it works nice on the first launch, but when the route changes it acts strangely

**Comment by ludovic-gonthier** [comment]
I managed to make the make the disabled attribute working fine. 
You can see the changes on my branch `responsive-navbar`.
If this is ok for you, push the changes to your branch and I'll merge the PR.

**Pull request merged**